docs/frontend_readme.md

```diff
@@ -85,12 +89,15 @@ Run the following on the host machine (not in the container):
    nvm install node && nvm alias default node
    ```
 
+1. Optional: add a `.env.local` file:
```

Comment:
Maybe add an explanation as to why.  Like if you'd like to override environment variables and then point them to the NX docs for further information? https://nx.dev/recipes/tips-n-tricks/define-environment-variables